backend/app/data_product_types/schema.py

```diff
@@ -1,11 +1,7 @@
 from uuid import UUID
 
-from app.data_product_types.enums import DataProductIconKey
 from app.data_product_types.schema_create import DataProductTypeCreate
-from app.data_products.schema import DataProduct
 
 
 class DataProductType(DataProductTypeCreate):
     id: UUID
-    data_products: list[DataProduct]
-    icon_key: DataProductIconKey
```

Comment:
why are both these fields removed? Are they not needed anymore? Should we also remove them from the table completely?